napari/_app_model/constants/_menus.py

```diff
@@ -90,6 +90,8 @@ def contributables(cls) -> set['MenuId']:
 class MenuGroup:
     NAVIGATION = 'navigation'  # always the first group in any menu
     RENDER = '1_render'
+    # View menu
+    ZOOM = 'zoom'
```

Comment:
Yeah ngl I have no idea what the structure of this `MenuGroup` class really is... should `Plugins` stuff actually be a subclass, like `LAYERLIST_CONTEXT`? And also `File` stuff? And then this would be the only group in the `View` subclass? Agreed we can leave as is for now but should probably decide and update docstring

Reply:
I just noticed the subclasses! I mean maybe that is what Talley had in mind as I see it was you and I that added the plugins and file menu bar ones, while the others sit within subclasses. I have no idea with 'RENDER' means though?

Reply:
>  I have no idea with 'RENDER' means though?

lol neither. But it seems like maybe the top level groups are maybe supposed to be used across other menus? Right now we use them in `View` menu and `Help` menu but I don't think that answers any questions about its semantics lol